backend/app/tasks/robot_queue_tasks.py

```
@@ -0,0 +1,903 @@
+    (its online key is `device:online:{owner}:{device_id}`), which may differ
+    from the robot creator in mixed dev environments.
+    """
+
+    from app.models.kind import Kind
+
+    row = (
+        db.query(Kind)
+        .filter(
+            Kind.kind == "Device",
+            Kind.name == device_id,
+            Kind.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)